tests/integration/cloud/clouds/test_joyent.py

```diff
@@ -5,95 +5,29 @@
 
 # Import Python Libs
 from __future__ import absolute_import, print_function, unicode_literals
-import os
 
 # Import Salt Testing Libs
-from tests.support.case import ShellCase
-from tests.support.paths import FILES
-from tests.support.helpers import expensiveTest, generate_random_name
-
-# Import Salt Libs
-from salt.config import cloud_providers_config
+from tests.support.unit import skipIf
 
 # Create the cloud instance name to be used throughout the tests
-INSTANCE_NAME = generate_random_name('CLOUD-TEST-')
+from tests.integration.cloud.helpers.cloud_test_base import CloudTest, TIMEOUT
+
 PROVIDER_NAME = 'joyent'
 
 
-class JoyentTest(ShellCase):
+@skipIf(True, 'Joyent is EOL as of November 9th, 2019.  It will no longer be supported in salt-cloud')
+class JoyentTest(CloudTest):
```

Comment:
I thought we agreed to remove these `joyent` tests.

Reply:
:+1: I'll delete the file